qnvim.pro

```diff
@@ -27,9 +27,9 @@ isEmpty(IDE_BUILD_TREE): IDE_BUILD_TREE = $$(QTC_BUILD)
 ##    "%LOCALAPPDATA%\QtProject\qtcreator" on Windows Vista and later
 ##    "$XDG_DATA_HOME/data/QtProject/qtcreator" or "~/.local/share/data/QtProject/qtcreator" on Linux
 ##    "~/Library/Application Support/QtProject/Qt Creator" on macOS
-USE_USER_DESTDIR = yes
+##USE_USER_DESTDIR = yes
```

Comment:
Is it required for arch linux build process? I think enabling `USE_USER_DESTDIR` has some benefits for normal users.

Reply:
I need this commented to create a package. I can also comment out this line using the package script, but in other distros it may be harder to workaround this. And your comment also says “uncomment to build plugin into user config directory”, so I thought it might be a typo.
So, I suggest to avoid to hardcode this option because it will disallow to override it from command line. User can always pass `USE_USER_DESTDIR=yes` to `qmake` or just edit this file.